### PR TITLE
airgap enablement: round 2

### PR DESCRIPTION
**Pull request by hdonnay**

### CODE REVIEW

**Comment by hdonnay** [comment]
>Does default need to be it's own package ? Could updater package just export this method?
Having the registry import the default set of updaters means those must be imported. Having it separated out means a package can use the registry without directly depending on any of the updaters.

>Do you feel strongly about implementing the registry as several package level functions working on a shared object? It would follow our pattern more to make a Registry struct with methods reflecting the package level functions, and optionally returning a "DefaultRegistry" instead of having to call "Set" before hand.
Yeah, I think having multiple registries defeats the point of using a registry. If you need to pass something in then just require it be configured beforehand.

>Having a registry also allows a nice place to take the config's "UpdaterSet" name and return the factory, should this be implemented on the registry as well?
Isn't this just a map access?

File: cmd/libvulnhttp/main.go [hunk 1]
```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/rs/zerolog/log"
 
 	"github.com/quay/claircore/libvuln"
+	"github.com/quay/claircore/updater/defaults"
```
Comment (ldelossa) [inline cmd/libvulnhttp/main.go:16]: underscore desired before path string ?

**Comment by ldelossa** [comment]
@hdonnay my only nit is considering a name for "defaults" package which makes it immediately clear it's for the registry in updater package.

maybe "updater/registrydefaults"

File: cmd/libvulnhttp/main.go (hunk 1 above)
Comment (hdonnay) [reply cmd/libvulnhttp/main.go:16]: I have the server explicitly check for errors down below, so it needs the name.

**Pull request merged**